neuralop/models/codano.py

```diff
@@ -0,0 +1,455 @@
+import torch.nn as nn
+import torch.nn.functional as F
+import torch
+import numpy as np
+from ..layers.channel_mlp import ChannelMLP
+from ..layers.spectral_convolution import SpectralConv
+from ..layers.skip_connections import skip_connection
+from ..layers.padding import DomainPadding
+from ..layers.coda_blocks import CODABlocks
+from ..layers.resample import resample
+from ..layers.embeddings import GridEmbedding2D, GridEmbeddingND
+
+class CODANO(nn.Module):
+
+    """ Codomain Attention Neural Operators (CoDA-NO) uses a specialized attention mechanism in the codomain space for data in 
+    infinite dimensional spaces as described in [1]_. The model treates each input channel as a variable of the pyscial system
+    and uses attention mechanism to model the interactions between the variables. The model uses lifting and projection modules
+    to map the input variables to a higher-dimensional space and then back to the output space. The model also supports positional
+    encoding and static channel information for additional context of the physical system such as external force or inlet condition.
+
+
+    Parameters
+    ----------
+    output_variable_codimension : int
+        The codomain dimension or the number of output channels corresponding to each input variable (or channel). Default is 1.
+        Example: For a input with 3 variables (channels) and output_variable_codimension=2, the output will have 6 channels (3 variables × 2 codimension).
+
+    lifting : bool
```

Comment:
`lifting_variable_codimension` and `hidden_variable_codimension` could be clarified, or the behavior should be explained in the case where `lifting=False`. I suggest changing the argument name to `lifting_codimension` or `lifting_channels` and allowing `None` instead of a separate bool option